proto/one/enterprise/data/report_definition_json_v2.proto

```diff
@@ -10,6 +10,7 @@ enum EnumReportDefinitionCategory {
     REPORT_DEFINITION_CATEGORY_SPREADSHEET = 1;
     REPORT_DEFINITION_CATEGORY_SAMPLE_MANAGER = 2;
     REPORT_DEFINITION_CATEGORY_TENANT = 3;
+	REPORT_DEFINITION_CATEGORY_AQTS_LICENSE = 4;
```

Comment:
```suggestion
    REPORT_DEFINITION_CATEGORY_AQTS_LICENSE = 4;
```

Reply:
Didn't realize this involved a protobuf change, is there further downstream changes required for this in terms of the Reporting MsApp or other places?

Said differently, is this enum value only needed by the frontend or is there places in the backend logic that need this as well?